src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -62,10 +63,10 @@ public ErrorResponse handleResourceExists(ResourceExistsException e) {
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT);
     }
 
-    @ExceptionHandler(OsiamBackendFailureException.class)
+    @ExceptionHandler({OsiamBackendFailureException.class, ConnectionInitializationException.class})
```

Comment:
Because that's what we agreed on: Generate the log message as close as possible to where the exception is generated but turn the exception into a scim error response in the exception handler.